base/MAPL_EsmfRegridder.F90

```diff
@@ -1337,6 +1388,12 @@ subroutine initialize_subclass(this, unusable, rc)
              & dynamicMaskRoutine=simpleDynMaskProcV, &
              & rc=rc)
         _VERIFY(rc)
+     case (REGRID_METHOD_BILINEAR_MONOTONIC, REGRID_METHOD_CONSERVE_MONOTONIC)
+        call ESMF_DynamicMaskSetR4R8R4V(this%dynamic_mask, &
+             & dynamicSrcMaskValue=MAPL_undef, &
+             & dynamicMaskRoutine=monotonicDynMaskProcV, &
+             & rc=rc)
+        _VERIFY(rc)
```

Comment:
@bena-nasa do you need to set handleAllElements=.true. ?

Reply:
Good catch there Rocky, was copying and pasting and forgot that, thanks!